CeeFM/Decideify
# DISCOGS API IS UP AND RUNNING

**Description**
Thank goodness the Discogs API is here to save me. You can easily search by genres and albums and get the returned data with image files included - MUCH easier than the MusicBrainz route of having to grab a musicbrainz ID for the album and do a separate fetch call (sometimes multiple fetch calls) to even check if an image file is available. DISCOGS THANK YOU SO MUCH I LOVE YOU
**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Go to the music content hub /music
- Click the "Test Music API" button, nothing should appear in the console
- Click the "Print Music" button, and an array of 100 objects should be printed to the console, each individual object representing a rock album from the discogs database. 
- Bask in the glory of having switched API's for your final capstone project and having it be only mildly painful to start! Hoorah this is what I want to do for a living!!! I'M BEING GENUINE AND SERIOUS!!!!!

Checklist
